### PR TITLE
[IMP] pos_sale: show unpaid amounts in the sale order list

### PR DESCRIPTION
**Pull request by caburj**
TASK-ID: 2895483

**NOTE:** It would be nice to use the utility classes from bootstrap 5 that is currently introduced in the backend UI. But pos is not loading bootstrap, so I'm just manually introducing "utility classes" in this PR.

**Normal View:**
Before:
image
After:
Screenshot 2022-07-12 at 15 34 06

**Mobile view:**
Before:
image
After:
image

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/pos_sale/static/src/css/pos_sale.css [hunk 1]
```diff
@@ -82,3 +82,21 @@
     background-color: #6EC89B;
     color: white;
 }
+
+/* Utility classes */
+
+.order-management-screen .flex {
```
Comment (pimodoo) [inline addons/pos_sale/static/src/css/pos_sale.css:88]: you already have .flex-container class

File: addons/pos_sale/models/sale_order.py [hunk 2]
```diff
@@ -26,6 +27,20 @@ def action_view_pos_order(self):
             'domain': [('id', 'in', linked_orders.ids)],
         }
 
+    @api.depends('invoice_ids.amount_total_signed', 'invoice_ids.state', 'pos_order_line_ids.price_subtotal_incl')
+    def _compute_amount_unpaid(self):
+        for sale_order in self:
```
Comment (pimodoo) [inline addons/pos_sale/models/sale_order.py:32]: here you should just do and it should do the trick and depends on the lines:

sale_order.amount_unpaid = sale_order.amount_total - sum(sale_order.order_line.filtered(lambda l: l.is_downpayment).price_total)

File: addons/pos_sale/models/sale_order.py [hunk 3]
```diff
@@ -26,6 +27,20 @@ def action_view_pos_order(self):
             'domain': [('id', 'in', linked_orders.ids)],
         }
 
+    @api.depends('invoice_ids.amount_total_signed', 'invoice_ids.state', 'pos_order_line_ids.price_subtotal_incl')
```
Comment (pimodoo) [inline addons/pos_sale/models/sale_order.py:30]: so depends should depend on order_line (I don't think we edit down_payment_lines, so no need of order_line.amount_total)

File: addons/pos_sale/models/sale_order.py [hunk 4]
```diff
@@ -26,6 +27,14 @@ def action_view_pos_order(self):
             'domain': [('id', 'in', linked_orders.ids)],
         }
 
+    @api.depends('order_line', 'amount_total', 'order_line.invoice_lines.move_id.state')
```
Comment (pimodoo) [inline addons/pos_sale/models/sale_order.py:30]: here it seems you can use "order_line.invoice_lines.parent_state" which is a related stored field

**Comment by pimodoo** [comment]
@robodoo r+ rebase-ff

**Comment by pimodoo** [comment]
@robodoo r-

**Comment by pimodoo** [comment]
can you squash when you have time? thanks

**Comment by caburj** [comment]
Done

**Comment by pimodoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**